cmd/kubelet/app/options/options.go

```diff
@@ -249,6 +243,28 @@ func NewKubeletConfiguration() (*kubeletconfig.KubeletConfiguration, error) {
 	return config, nil
 }
 
+// NewKubeletInstanceConfiguration will create a new KubeletInstanceConfiguration with
+// KubeletConfiguration shared as default values
+func NewKubeletInstanceConfiguration(kubeletConfig *kubeletconfig.KubeletConfiguration) (*kubeletconfig.KubeletInstanceConfiguration, error) {
+	scheme, _, err := kubeletscheme.NewSchemeAndCodecs()
+	if err != nil {
+		return nil, err
+	}
+
+	versionedInstance := &v1beta1.KubeletInstanceConfiguration{}
+	configInstance := &kubeletconfig.KubeletInstanceConfiguration{}
+	if err := scheme.Convert(versionedInstance, configInstance, nil); err != nil {
+		return nil, err
+	}
+
+	// Initialize the KubeletInstanceConfig as default with KubeletConfiguration values
```

Comment:
This seems a good point to overwrite the `default` internal values of the `KubeletInstanceConfiguration`, being them the original values of `KubeletConfiguration`. Some ideas to improve it:

1. API machinery could have a `scheme.Default(versioned)` like or has some better alternative them manually do it one by one.
2. Use reflection to "automatically" fill the fields.